msal/tests/dev apps/XForms/XForms/AccessTokenCacheItemDetails.xaml.cs

```diff
@@ -52,7 +53,7 @@ public partial class AccessTokenCacheItemDetails : ContentPage
             expiresOnLabel.Text = msalAccessTokenCacheItem.ExpiresOn.ToString();
             scopesLabel.Text = msalAccessTokenCacheItem.NormalizedScopes;
 
-            cachedAtLabel.Text = CoreHelpers.UnixTimestampToDateTime(msalAccessTokenCacheItem.CachedAt).ToString();
+            cachedAtLabel.Text = CoreHelpers.UnixTimestampToDateTime(Convert.ToInt64(msalAccessTokenCacheItem.CachedAt)).ToString();
```

Comment:
>Convert.ToInt64( [](start = 69, length = 16)

can this be moved to the MsalAccessTokenCacheItem class?

Reply:
yes, will change